app/screens/image_preview/downloader.ios.js

```diff
@@ -192,29 +299,55 @@ export default class Downloader extends PureComponent {
                             id='mobile.downloader.downloading'
                             defaultMessage='Downloading...'
                             style={styles.bottomText}
-                        /> :
-                        <FormattedText
-                            id='mobile.downloader.success'
-                            defaultMessage='Image Saved'
-                            style={styles.bottomText}
-                        />
+                        /> : savedComponent
```

Comment:
Can we get rid of the ternary statement here and combine it with the logic you added above? This is more complexity than we want inside a ternary